tailwind.config.cjs

```diff
@@ -43,6 +43,23 @@ module.exports = {
           'Times',
           'serif',
         ],
+        altSans: [
```

Comment:
surprised to see a second sans-serif font. to make clear that the fallback is our regular sans, please define this in terms of the other. e.g.
```
altSans: [
  'Mulish',
  ...sans,
]
```

Reply:
Yea but it does look a lot nicer on the buttons for that small text. And done thanks!